Cannot execute helloworld-sample test: error ENOTFOUND update.code.visualstudio.com at setup stage

I have some issues to use the helloworld-sample test supplied in vscode-extension-tester:

1. After "npm install" in the vscode-extension-tester/sample-projects/helloworld-sample folder, "npm run compile" gives multiple errors like:
error TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i --save-dev @types/jest` or `npm i --save-dev @types/mocha`.
=> Adding "@types/mocha": "^9.0.0" to the "devDependencies" in package.json solves this first issue

2. "npm run ui-test" fails with the following error:
RequestError: getaddrinfo ENOTFOUND update.code.visualstudio.com
    at ClientRequest.<anonymous> (.../vscode-extension-tester/sample-projects/helloworld-sample/node_modules/got/dist/source/core/index.js:962:111)
    at Object.onceWrapper (node:events:510:26)
    at ClientRequest.emit (node:events:402:35)
    at ClientRequest.origin.emit (.../vscode-extension-tester/sample-projects/helloworld-sample/node_modules/@szmarczak/http-timer/dist/source/index.js:43:20)
    at TLSSocket.socketErrorListener (node:_http_client:447:9)
    at TLSSocket.emit (node:events:390:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26)

My environment is as follow:
- Linux RHEL7
- node-linux-x64-16.13.0
- VSCode-linux-x64-1.62.3

Could you please help to understand/solve this issue ?
Thanks !

--- a/package.json
+++ b/package.json
@@ -42,6 +42,7 @@
   "devDependencies": {
     "@types/fs-extra": "^9.0.12",
     "@types/glob": "^7.1.4",
+    "@types/global-agent": "^2.1.1",
     "@types/js-yaml": "^4.0.2",
     "@types/mocha": "^8.2.3",
     "@types/node": "^14.11.2",
@@ -56,6 +57,7 @@
     "compare-versions": "^3.6.0",
     "fs-extra": "^10.0.0",
     "glob": "^7.1.7",
+    "global-agent": "^3.0.0",
     "got": "^11.8.2",
     "js-yaml": "^4.1.0",
     "monaco-page-objects": "^1.9.0",

--- a/src/util/download.ts
+++ b/src/util/download.ts
@@ -2,6 +2,7 @@ import * as fs from 'fs-extra';
 import got from 'got';
 import { promisify } from 'util';
 import stream = require('stream');
+import  'global-agent/bootstrap';
 
 export class Download {
     static async getText(uri: string): Promise<string> {

